client/gutenberg/extensions/tiled-gallery/layout/index.js

```diff
@@ -97,3 +105,33 @@ export default class Layout extends Component {
 function isSquareishLayout( layout ) {
 	return [ 'circle', 'square' ].includes( layout );
 }
+
+function isWpcomFilesUrl( url ) {
+	const { host } = parseUrl( url );
+	return /\.files\.wordpress\.com$/.test( host );
+}
+
+function photonWpcomImage( url, opts ) {
+	// Adhere to the same API as the photon.js lib
```

Comment:
Should we explain more why we're doing this (instead of just using `Photon.js`), so that anyone new to this block will understand the "_Why_" better?

Reply:
Why as in "above this is used as a drop-in replacement for the photon library" or why as in "WPCOM images on private sites blah blah…"?